### DISCUSSION TITLE
Convert dataset to csv or dataframe

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#2444] [Q&A] [answered]
Hi,
Is there a way to convert a burn dataset to csv or dataframe ? If so, I can save the inference results to the disk.

**Comment by laggui** [comment, marked as answer]
There is no direct path to save a Burn dataset to csv or dataframe. To save your results it should be pretty straightforward to convert the values to vec or a similar data structure and save them using the [`csv` crate](https://docs.rs/csv/latest/csv/index.html).
